src/main/resources/assets/corpse/lang/ru_ru.json

```diff
@@ -1,8 +1,6 @@
 {
   "entity.corpse.corpse": "Труп",
   "entity.corpse.corpse_of": "Труп %s",
-  "button.corpse.previous": "<",
-  "button.corpse.next": ">",
```

Comment:
Why remove this?

Reply:
Because I'm sure that these two lines can be normally translated into Russian (look into the Russian translation file from your GraveStone mod, for example).

Reply:
I think a universal symbol like an arrow is always better than a language that somebody might not understand.

Reply:
I also think the person that translated the gravestone mod abbreviated the translation, because it was too long for the buttons. This was also the reason why the person that translated this used the arrows.
So we could use the translation of the gravestone mod:

> "button.gravestone.next": "След.",
  "button.gravestone.prev": "Пред."

Reply:
Fair enough. Do you think we should use them here as well? I don't know Russian, so I'm not sure whether they are translated and abbreviated correctly or not :/

Reply:
I put them into a translator. Seems to be correct. If you add this to your PR, I will merge it.

Reply:
Done.